composer.json

```diff
@@ -37,17 +37,17 @@
     ],
 
     "require": {
-        "php": ">=5.5",
-        "league/climate": "^2.0|^3.0",
+        "php": "^5.6 || ^7.0",
+        "league/climate": "^2.0 || ^3.0",
         "symfony/console": "^2.0",
         "symfony/process": "^2.1"
     },
 
     "require-dev": {
         "mockery/mockery": "^0.9",
-        "phpunit/phpunit": "^4.0",
+        "phpunit/phpunit": "^5.1",
```

Comment:
Disagree here, this is a dev dependency, not installed if static-review is itself a dependency.

Worth sticking to the same major version in development rather that supporting two in my view.